src/Microsoft.VisualStudio.Editors/Common/SplitButton.vb

```diff
@@ -29,7 +30,7 @@ Namespace Microsoft.VisualStudio.Editors.Common
             MyBase.New()
 
             AccessibleRole = AccessibleRole.SplitButton
-            _pushButtonWidth = DpiHelper.LogicalToDeviceUnitsX(_pushButtonWidth)
+            _pushButtonWidth = DpiAwareness.LogicalToDeviceUnits(Handle, _pushButtonWidth)
```

Comment:
Can you point me to docs here? I want to know why the Handle is interesting. For example, here, this control isn't actually visible anywhere and you are force loading the Handle (I think it gets added to a hidden window) so might not work as expected.

Reply:
I forwarded you the *Impactful change notification* that went out.  Don't know if there are other docs here.

Reply:
I've asked offline to make sure this works as expected.

Reply:
Conclusion internally:

> None of our WinForms code should be running in PMA, only System. Which means relying on the Handle before parented should be fine, assuming we wrap dialogs in [a SystemAware scope](https://github.com/dotnet/project-system/issues/4493). This means we don’t need to react to DPI changes either.

@etbyrd Can we make sure that that all the UI we’re touching is in System-Aware DPI mode via winspy++? Drag the target to the control and then click the DPI tab.